defaults/main.yml

```diff
@@ -0,0 +1,26 @@
+LOAD_BALANCER_SERVER_IP: "{{ lookup('dig', ansible_host) }}"
```

Comment:
@bdero Yes, I've already done this locally, but thanks for reminding me.  It will be included in the PR with the actual playbook.